cranelift/unwinder/Cargo.toml

```diff
@@ -0,0 +1,33 @@
+[package]
+authors = ["The Cranelift Project Developers"]
+name = "cranelift-unwinder"
+version = "0.121.0"
+description = "Unwinder that works with Cranelift's exception support"
+license = "Apache-2.0 WITH LLVM-exception"
+documentation = "https://docs.rs/cranelift-unwinder"
+repository = "https://github.com/bytecodealliance/wasmtime"
+categories = ["no-std"]
+readme = "README.md"
+keywords = ["compile", "compiler", "jit"]
+edition.workspace = true
+rust-version.workspace = true
+
+[lints]
+workspace = true
+
+[dependencies]
+cranelift-codegen = { workspace = true, optional = true }
+log = { workspace = true }
+cfg-if = { workspace = true }
+object = { workspace = true }
+anyhow = { workspace = true }
+
+[target.'cfg(target_arch = "s390x")'.dependencies]
+psm = { workspace = true }
```

Comment:
FWIW if you're feeling enterprising IIRC the only reason we have this is `get_stack_pointer` on s390x where inline assembly wasn't supported originally, but it is supported now, so a small inline assembly snippet would probably suffice for removing this crate dependency.

Reply:
Done!